pages.ru/common/hexdump.md

```diff
@@ -0,0 +1,20 @@
+# hexdump
+
+> Дамп файла в ASCII, десятичном, шестнадцатеричном и восьмеричном форматах.
+> Больше информации: <https://manned.org/hexdump>.
+
+- Распечатать шестнадцатеричное представление файла, заменяя повторяющиеся строки на '*':
+
+`hexdump {{файл}}`
+
+- Отобразить шестнадцатеричное и ASCII представление в две колонки:
+
+`hexdump -C {{файл}}`
```

Comment:
Syntax is incorrect both in the original page and here. Is should be (it's assumed that path to file is expected and not just a plain filename):

```suggestion
`hexdump -C {{путь/к/файлу}}`
```